### PR TITLE
Add closing procedure for mainline BZ to Bug-report-Life-Cycle.md

### PR DESCRIPTION
**Pull request by anoopcs9**
Signed-off-by: Anoop C S <anoopcs@redhat.com>

### CODE REVIEW

File: Contributors-Guide/Bug-report-Life-Cycle.md [hunk 1]
```diff
@@ -36,6 +36,8 @@ This page describes the life of a bug report.
     -   **CLOSED/CURRENTRELEASE** when a code change that fixes the
         reported problem has been merged in
         [Gerrit](http://review.gluster.org).
+    -   **CLOSED/NEXTRELEASE** when backport of the code change the
```
Comment (atinmu) [inline Contributors-Guide/Bug-report-Life-Cycle.md]: s/the/that ?

Comment (anoopcs9) [inline Contributors-Guide/Bug-report-Life-Cycle.md]: Yup.

Comment (anoopcs9) [inline Contributors-Guide/Bug-report-Life-Cycle.md]: Please see the updated PR

**Comment by atinmu** [comment]
Looks good, I don't have merge privileges  :)

File: Contributors-Guide/Bug-report-Life-Cycle.md [hunk 2]
```diff
@@ -36,6 +36,8 @@ This page describes the life of a bug report.
     -   **CLOSED/CURRENTRELEASE** when a code change that fixes the
```
Comment (nixpanic) [inline Contributors-Guide/Bug-report-Life-Cycle.md]: We only use CLOSED/CURRENTRELEASE when the patch is merged in the release-branch for the version that this bug is reported against, **and** a version (tagged) is released with the fix.

Could you update this as well?

Comment (anoopcs9) [inline Contributors-Guide/Bug-report-Life-Cycle.md]: Sure.

**Comment by prashanthpai** [comment]
Review comments seems to be addressed. Merging it.

**Pull request merged**